content/06.resources.md

```diff
@@ -0,0 +1,12 @@
+
+### Data and Resource availability
+
+This manuscript was written using Manubot [@doi:10.1371/journal.pcbi.1007128] and is available on github: [manuscript repository link](https://github.com/greenelab/nature_news_manuscript). 
+All code and metadata is also available on github, [full analysis repository link](https://github.com/greenelab/nature_news_disparities), under a BSD 3-Clause License.
+The code to generate all main and supplemental figures are available as R markdown documents within our main analysis github, in the following subfolder: [notebooks](https://github.com/greenelab/nature_news_disparities/tree/main/figure_notebooks).
+Due to copyright, we are unable to provide the scraped data used in this analysis.
+However, scraping code is available on our main analysis github, in the following subfolder: [scraper](https://github.com/greenelab/nature_news_disparities/tree/main/nature_news_scraper).
+
+
+### Acknowledgements
+We would like to thank Jeffrey Perkel for spurring this line of research and providing thoughtful feedback throughout the course of this project. 
```

Comment:
We need to ask Jeff if we can acknowledge him before we add this. Would you send him an e-mail asking if it's ok?
```suggestion
We would like to thank Jeffrey Perkel for asking thoughtful questions that spurred this line of research and providing feedback and insight into the news-gathering process during the course of this project. 
```

Reply:
while we wait, I will comment it out for now.
commented out in: https://github.com/greenelab/nature_news_manuscript/pull/2/commits/cea4082f2927ad1ac7a30f8b88ea5ab969a0f819